src/components/HookForm/index.jsx

```diff
@@ -168,6 +168,12 @@ export default class HookForm extends Component {
       scheduleTextField: '',
       taskValidJson: true,
       triggerSchemaValidJson: true,
+      validation: {
+        owner:  {
+          error: false,
+          message: "",
```

Comment:
We use single quotes for strings and double quotes for jsx. Use single quotes here.